cli/command/password/change.go

```diff
@@ -0,0 +1,59 @@
+package password
+
+import (
+	"github.com/appcelerator/amp/api/rpc/account"
+	"github.com/appcelerator/amp/cli"
+	"github.com/spf13/cobra"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+)
+
+type changeOpts struct {
+	current_password string
+	new_password     string
+}
+
+var (
+	changeOptions = &changeOpts{}
+)
+
+// NewChangeCommand returns a new instance of the change command.
+func NewChangeCommand(c cli.Interface) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "change",
+		Short: "Change password",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return change(c, cmd)
```

Comment:
you could add 
PreRunE: cli.NoArgs,
considering this command doesn't have argument anf get an automatic error if an argument is set